draft-ietf-sedate-datetime-extended.md

```diff
@@ -401,27 +353,25 @@ account.
 implementations that they should project it to the Hebrew calendar.
 
 ~~~~
-1996-12-19T16:39:57-08:00[x-foo=bar][x-baz=bat]
+1996-12-19T16:39:57-08:00[_foo=bar][_baz=bat]
 ~~~~
-{: #date-time-private title="Adding tags in private use namespaces"}
+{: #date-time-private title="Adding experimental tags"}
 
-{{date-time-private}}, based on {{rfc3339-datetime}}, utilizes the private use namespace to declare two
-additional pieces of information in the suffix that can be interpreted
-by any compatible implementations and ignored otherwise.
+{{date-time-private}}, based on {{rfc3339-datetime}}, utilizes keys
+identified as experimental by a leading underscore to declare two additional pieces of
+information in the suffix; these can be interpreted by implementations
+that take part in the controlled experiment making use of these tag keys.
 
 # IANA Considerations {#iana-cons}
 
 IANA is requested to establish a registry called "Timestamp Suffix Tag Keys".
 Each entry in the registry shall consist of the information described in {{registered}}.
-Initial contents of the registry are specified in {{namespaced}}.
-
-The policy is "RFC required", "Specification Required", ???[^policy]
-{{RFC8126}}.
-
-[^policy]: We need to define the policy for both namespaces and full keys.
-{: source="--- cabo"}
-
+Initial contents of the registry are specified in {{registered}}.
```

Comment:
Is it intentional for the referenced section to not actually define any tag keys, even the "u-ca" used in the third example?

Reply:
No, this is a gaping hole. Added a TODO in the text.